js/components/scrubberBar.jsx

```diff
@@ -507,6 +572,11 @@ class ScrubberBar extends React.Component {
         onMouseLeave={this.handleScrubberBarMouseLeave}
         onMouseMove={scrubberBarMouseMove}
       >
+        {controller.state.markers && controller.state.markers.list.length > 0 && (
```

Comment:
`controller.state.markers` will be rendered instead of div if `controller.state.markers.list.length <= 0`. Convert it into boolean: `{!!controller.state.markers && controller.state.markers.list.length > 0 && (`